aws-vpcflow/package.yaml

```
@@ -0,0 +1,116 @@
+id: aws-vpcflow
+name: AWS VPC Flow Logs
+author: Tenzir
+author_icon: https://raw.githubusercontent.com/tenzir/library/main/author.svg
+package_icon: https://raw.githubusercontent.com/tenzir/library/main/aws-vpcflow/package.svg
+
+description: |
+  AWS VPC Flow Logs capture information about IP traffic going to and from
+  network interfaces in your VPC. This package provides utilities for
+  ingesting, parsing, and analyzing VPC Flow Log data.
+  
+  This package provides:
+  - Raw VPC Flow Log events on the 'aws' topic as 'aws.vpc_flow' events
+  - OCSF-formatted network activity events on the 'ocsf' topic as 'ocsf.network_activity' events
+  
+  To use this package, you must configure your AWS VPC to log to S3 in text
+  format as described in the AWS documentation:
+  https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs-s3.html
+
+categories:
+- connectors
+
+inputs:
+  s3-bucket-url:
+    name: S3 Bucket URL
+    description: |
+      The S3 URL path containing VPC Flow Log files.
+    default: "s3://your-bucket/AWSLogs/**.log.gz"
+  remove-inputs:
+    name: Remove Files After Importing
+    description: |
+      Whether to remove files after processing them. Possible values are 'true' or 'false'.
+    default: "false"
+
+contexts:
+
+pipelines:
+  ingest-from-s3:
+    name: Ingest VPC Flow Logs from S3
+    description: |
+      Ingests VPC Flow Log files from an S3 bucket and publishes to 'aws' topic.
+    definition: |
+      from_file "{{ inputs.s3-bucket-url }}", watch=true, remove={{ inputs.remove-inputs }} {
+        decompress_gzip
+        read_xsv field_separator=" ", list_separator=",", null_value="-"
+      }
+      @name = "aws.vpc_flow"
+      publish "aws"
```

```suggestion
      @name = "amazon.vpc_flow"
      publish "amazon"
```

Most Amazon products now have the "Amazon" prefix. Should we therefore use that here (and below) as well?